src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,130 @@
 package core.basesyntax;
 
+import java.util.NoSuchElementException;
+
 public class ArrayList<T> implements List<T> {
+    private static final int DEFAULT_CAPACITY_FOR_ARRAY = 10;
+    private static final String INDEX_ERROR_MESSAGE =
+            "The passed index is not allowed";
+    private static final String ELEMENT_NOT_FOUND_MESSAGE = "Element not found";
+    private Object[] elementData;
```

Comment:
Do not use class Objects, you have to use generic

Reply:
Thank you for your review, but I use Object which is not the same as Objects. I chose Object because it is used in the standard ArrayList